dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -894,7 +922,53 @@ export const Explorer = ({
 
   const handleQueryChange = async (newQuery: string) => setTempQuery(newQuery);
 
+  // need to move to common , copied from explorer
```

Comment:
Add TODO here. TODOs are marked separately in the code editor and can be identifiable.